central/delegatedregistryconfig/delegator/delegator.go

```diff
@@ -134,6 +141,64 @@ func (d *delegatorImpl) DelegateScanImage(ctx context.Context, imgName *storage.
 	return image, nil
 }
 
+// DelegateScanImageV2 sends a scan request to the provided cluster.
+func (d *delegatorImpl) DelegateScanImageV2(ctx context.Context, imgName *storage.ImageName, clusterID string, namespace string, force bool) (*storage.ImageV2, error) {
+	if !features.FlattenImageData.Enabled() {
+		return nil, errors.New("ROX_FLATTEN_IMAGE_DATA flag is not enabled")
```

Comment:
nit: you can use `features.FlattenImageData.EnvVar()` instead of writing `ROX_FLATTEN_IMAGE_DATA` in this error message.